Monika After Story/game/script-topics.rpy

```diff
@@ -6288,6 +6288,114 @@ label monika_yellowwp:
             m 1a "It'll definitely be an interesting read for you."
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_driving",category=['monika'],prompt="Can you drive?",random=True))
+
+label monika_driving:
+    m 1d "Hm? Can I drive?"
+    m 1c "I never really thought about getting a driver's licence."
+    m 3a "Public transportation is enough for me usually."
```

Comment:
correction "i usually use public transportation as its pretty convenient and other times i just walk"